frontend/containers/open-call-page/chart/component.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+
+import { Doughnut } from 'react-chartjs-2';
+
+import { Chart as ChartJS, ArcElement } from 'chart.js';
+
+import { OpenCallChartProps } from '.';
+
+ChartJS.register(ArcElement);
+
+export const OpenCallChart: FC<OpenCallChartProps> = ({ openCallRange }) => {
```

Comment:
Two comments:
1. There's a weird hover effect. I think you could remove it entirely.
2. This is supposed to be a progress bar. The example you gave me has a deadline in October so I think the ring shouldn't be entirely filled. 0% should be the first day of the launch, 100% the last day before the closure.

Reply:
The idea is that the green color means 'available days to apply' and the green color means 'consumed days' 
So maybe we should add a label to explain that